helm/hpcc/values.yaml

```diff
@@ -234,70 +234,70 @@ bundles: []
 dali:
 - name: mydali
   services: # internal house keeping services
-  - service: coalescer
-    #servicePort: 8877
-    #interval: 2 # (hours)
-    #at: "* * * * *" # cron type schedule, i.e. Min(0-59) Hour(0-23) DayOfMonth(1-31) Month(1-12) DayOfWeek(0-6)
-    #minDeltaSize: 50 # (Kb) will not start coalescing until delta log is above this threshold
-    #recoverFromIncErrors: false # Try to continue after transaction load failures
-    #resources:
-    #  cpu: "1"
-    #  memory: "1G"
-
-  # metaAccess: "shared" # enable if coalescer running in separate pod, and thus needs access to meta over PVC mount
+    coalescer:
+      #servicePort: 8877
+      #interval: 2 # (hours)
+      #at: "* * * * *" # cron type schedule, i.e. Min(0-59) Hour(0-23) DayOfMonth(1-31) Month(1-12) DayOfWeek(0-6)
+      #minDeltaSize: 50 # (Kb) will not start coalescing until delta log is above this threshold
+      #recoverFromIncErrors: false # Try to continue after transaction load failures
```

Comment:
recoverFromIncErrors is not defined in the schema.

Reply:
I didn't mean for this to be here, will remove.